components/application/widgets/SingleItem.js

```diff
@@ -107,20 +114,24 @@ export function SingleItems({
     } catch (error) {}
   };
 
-  const deleteTaskHandler = async (task) => {
+  const deleteTaskHandler = async () => {
     try {
       const res = await fetch("/api/deleteTask", {
         method: "POST",
         headers: {
           "Content-Type": "application/json",
         },
-        body: JSON.stringify(task),
+        body: JSON.stringify({ _id: _id }),
       });
 
       const result = await res.json();
       console.log(result);
     } catch (error) {}
```

Comment:
Once again, it seems like there hasn't been a decision on how to handle errors. If that's the case, feel free to disregard these two comments. Just a reminder to keep track for future iterations and decide on the error handling approach.

Reply:
fixed. These two function writen by Kelvin, I haven't noticed.